src/Tests/Project/JobRunner.test.ts

```diff
@@ -0,0 +1,46 @@
+/*
+ * Copyright (c) 2022 Samsung Electronics Co., Ltd. All Rights Reserved
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import {assert} from 'chai';
+
+import {JobRunner} from '../../Project/JobRunner';
+import {WorkJobs} from '../../Project/WorkJobs';
+import {obtainWorkspaceRoot} from '../../Utils/Helpers';
+import {Logger} from '../../Utils/Logger';
+import {MockJob} from '../MockJob';
+
+suite('Project', function() {
+  suite('JobRunner', function() {
+    const logger = new Logger();
+    suite('#start()', function() {
+      test('jobs are done', function(done) {
+        let jobRunner = new JobRunner(logger);
+        const workspaceRoot: string = obtainWorkspaceRoot();
+        let workJobs = new WorkJobs();
+        workJobs.push(new MockJob('mockup'));
+        workJobs.push(new MockJob('mockup'));
+        assert.strictEqual(workJobs.length, 2);
+
+        // overwrite events so that multiple events will be emitted
+        jobRunner.on('cleanup', function() {
+          assert.strictEqual(workJobs.length, 0);
+          done();
```

Comment:
Just curiosity, what does `done()` do?

Reply:
`done()` is a method to notify that this promiss function is done.

Reply:
Could you let me know what is promiss function in this context? Becuase IIUC, "promise" that I know is an `Promise` object. 

Well, what I found out is [this section](https://mochajs.org/#asynchronous-code) in mocha official web. 

> By adding an argument (usually named `done`) to `it()` to a test callback, Mocha will know that it should wait for this function to be called to complete the test.

So, `done` is just an aribtrary name of that exmaple and the name also should be the argument of `it()`. It seems the line should be `jobRunner.on('cleanup', function(done) {`. But you posted this PR after passing all tests.. Please let me know if I am wrong.

Reply:
`done()` is from `chai`

Reply:
Oh, I missed that,,